dhis-2/dhis-api/src/main/java/org/hisp/dhis/analytics/common/dimension/DimensionParamItem.java

```diff
@@ -81,13 +81,9 @@ public static List<DimensionParamItem> ofStrings( List<String> items )
     private static QueryOperator getOperator( String operator )
     {
         return Arrays.stream( QueryOperator.values() )
-            .filter( queryOperator -> queryOperator.name().equals( operator ) )
+            .filter( queryOperator -> equalsIgnoreCase( queryOperator.name(), operator ) )
             .findFirst()
-            .orElseThrow( IllegalAccessError::new );
+            .orElseThrow( () -> new IllegalArgumentException( "Invalid operator: " + operator ) );
```

Comment:
I believe we could throw something like `new IllegalQueryException( ErrorCode.E2035 );`
This should be automatically handled by the controller layer.